app/models/katello/system.rb

```diff
@@ -69,16 +62,18 @@ class System < Katello::Model
     before_update :update_foreman_host, :if => proc { |r| r.environment_id_changed? || r.content_view_id_changed? }
 
     scope :in_environment, ->(env) { where('environment_id = ?', env) unless env.nil? }
-    scope :completer_scope, ->(options) { readable(options[:organization_id]) }
     scope :by_uuids, ->(uuids) { where(:uuid => uuids) }
 
     scoped_search :on => :name, :complete_value => true
-    scoped_search :in => :environment, :on => :organization_id, :complete_value => true, :rename => :organization_id
+    scoped_search :in => :activation_keys, :on => :name, :complete_value => true, :rename => :activation_key
+    scoped_search :in => :content_view, :on => :name, :complete_value => true, :rename => :content_view
```

Comment:
Description would be nice as well -- some people use it as a notes field for marking content hosts